torch/_dynamo/variables/functions.py

```diff
@@ -1155,3 +1158,62 @@ def specialize_symbolic(self, arg: Any) -> Any:
         if isinstance(arg, SymNodeVariable):
             return ConstantVariable.create(arg.evaluate_expr())
         return arg
+
+
+class TMADescriptorVariable(VariableTracker):
```

Comment:
Needs a reconstruct method as well (the reconstruct is used to gracefully reconstruct the variable during a graph break)